### ISSUE TITLE
wine-tkg: version naming issue, "warning: tag 'v4.14-1' is really 'v4.15-1' here"

### ISSUE THREAD

**Issue by telans** [Tk-Glitch/PKGBUILDS#297]
Building either proton-tkg or wine-tkg (clean directory) return a version name of `4.14.1...` where it should be `4.15...`.

`[telans@desktop wine-staging-git]$ git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g;s/^v//;s/\.rc/rc/;s/^wine\.//'`
`warning: tag 'v4.14-1' is really 'v4.15-1' here`
`4.14.1.r4.g73491829`

**Comment by Tk-Glitch** [comment]
Yeah I should make it a PSA or something. Alistair frogged up and pushed the wrong tag mistakenly, then renamed it. That last part apparently creates issues https://stackoverflow.com/questions/14380053/renaming-git-tags-results-in-inconsistency
Fixing that correctly seems to require force-pushing on the wine-staging repo side. So it's kinda unwanted and might stay that way until the next tag is pushed for 4.16.
It's only cosmetic though, and the correct commit is still selected.
